mobile-widgets/qml/HintsTextEdit.qml

```diff
@@ -14,7 +14,11 @@ TextField {
 	onTextChanged: {
 		textUpdateTimer.restart();
 	}
-	onFocusChanged: frame.shouldShow = focus
+	onFocusChanged: {
+		focus != focus
```

Comment:
I'm by no means a javascript expert... but I think this isn't an assignment but a comparison.
Did you mean focus = !focus ?